ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthGetWork.java

```diff
@@ -33,6 +33,8 @@
 
   private final MiningCoordinator miner;
   private static final Logger LOG = getLogger();
+  private Boolean solverWasPresent = false;
+  private String[] storedResult;
```

Comment:
Don't store the stored result in the RPC method, query it from the mining co-ordinator.

Reply:
Mining coordinator doesn't store previous results. In-between results, there would be cases where `miner.getWorkDefinition().isPresent()` would return false, returning `JsonRpcError.NO_MINING_WORK_FOUND`. This would cause EthMiner connections to abort upon connecting for a few seconds.